grid.css

```diff
@@ -4,15 +4,15 @@ heading1, heading2{
 	width: 100%;
 	display: grid;
     grid-gap: 2%;	
-	grid-template-columns: 4% 26% 14% 22% 10% 7% 10%;
+	grid-template-columns: 4% 24% 14% 22% 11% 8% 10%;
```

Comment:
I adjusted the grid layout to fit the text better and give more space to those that need it and less to those that don't